dist/stylesheets/style.css

```diff
@@ -14,45 +16,135 @@ nav {
   justify-content: space-between;
   padding: 20px;
   background: #ffa9a3;
-  transition: 300ms; }
```

Comment:
All this should be auto-generated when saving the src/stylesheets/style.scss file. Do it like this.

Reply:
My style.css file has that row, pls check 900af5f3f31e88662b4ab0d62298569fa5787fed.